inbm/telemetry-agent/telemetry/telemetry_handling.py

```diff
@@ -303,7 +303,7 @@ def get_query_related_info(option: str, info: Dict) -> Dict:
                 del info['values'][elem]
     elif option == "version":
         info = {}
-        info['INBMVersionCommit'] = get_friendly_inbm_version_commit()        
+        info['INBMVersionCommit'] = get_friendly_inbm_version_commit()
     elif option == "all":
         info['INBMVersionCommit'] = get_friendly_inbm_version_commit()
```

Comment:
**<h2>$\color{#FFA500}Improvements: CodeSimplification$</h2>**
**Suggestion:** Refactor the get_query_related_info function to avoid direct deletion of keys in the loop, which can be error-prone and hard to read. Instead, use dictionary comprehension to filter out the keys.
**NOTE:** Ensure that the suggested code is comprehensive. This serves as an example of the code you could use. The user should verify it after accepting the suggestion.
```suggestion
    elif option == "version":
        return {'INBMVersionCommit': get_friendly_inbm_version_commit()}
    elif option == "all":
        info['INBMVersionCommit'] = get_friendly_inbm_version_commit()
    return info
```

Reply:
good suggestion but out of scope